scripts/linters/pylint_extensions.py

```diff
@@ -1469,6 +1475,93 @@ def process_module(self, node):
                     line=closing_line_index_of_fileoverview + 1)
 
 
+class NewlineBelowClassDocstring(checkers.BaseChecker):
+    """Checks if there is a single newline below the class docstring."""
+
+    __implements__ = interfaces.IRawChecker
+    name = 'newline-below-class-docstring'
+    priority = -1
+    msgs = {
+        'C0026': (
+            'Missing single newline below class docstring.',
+            'newline-below-class-docstring',
+            'Please add a single newline below class docstring.'
+        )
+    }
+
+    def process_module(self, node):
+        """Process a module to ensure that there is a single newline below
+        class docstring.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node to access module content.
+        """
+
+        in_class = False
+        in_docstring = False
+        class_has_docstring = False
+        file_content = read_from_node(node)
+        blank_line_counter = 0
+        prev_line = ''
+
+        for line_num, line in enumerate(file_content):
+            line = file_content[line_num].strip()
+
+            if line_num > 0:
+                prev_line = file_content[line_num - 1].strip()
+
+            # Check if it is a docstring and not some multi-line string.
+            if prev_line.startswith(b'class ') or in_class:
+                in_class = True
+                # Check if a class has docstring or not.
+                if prev_line.endswith(b'):'):
+                    in_class = False
+                    if line.startswith(b'"""'):
+                        in_docstring = True
+                        class_has_docstring = True
+
+            # This line is checking if docstring ends.
+            if line.endswith(b'"""'):
+                in_docstring = False
+                in_class = False
+                continue
+
+            # This line is checking if we are in a docstring.
+            if in_docstring:
+                continue
+
+            if class_has_docstring:
+                # This line is counting the empty lines.
+                if file_content[line_num] == b'\n':
+                    blank_line_counter += 1
+
+                # This line check if a class has no body if that is the case
+                # then skip check for this class. @ here is for the decorator.
+                elif (
+                        line.startswith(('class ', 'def', '@')) and
+                        not blank_line_counter == 0):
+                    blank_line_counter = 0
+                    class_has_docstring = False
+
+                # This line check if we reached to a non-empty line and
```

Comment:
Change it to `Checks if we reached to a non-empty line. If that's true it sets the variable blank_line_counter to zero and sets the variable class_has_docstring to false so that we can skip further lines until we find a next class.`

Reply:
Done.